sw/device/silicon_creator/lib/BUILD

```diff
@@ -327,16 +327,35 @@ cc_test(
     ],
 )
 
+cc_library(
```

Comment:
nit: not sure why you need this. using an `#ifdef OT_PLATFORM_RV32` should be enough, wasn't it?

Reply:
(N/a after removing the pure C implementation of CRC32.)